catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -724,59 +823,56 @@ interface PackageCreationDialogUIOptions {
 }
 
 interface UsePackageCreationDialogProps {
```

Comment:
this should be renamed to `PackageCreationDialogProps`, since it's no longer related to a hook